src/components/LanguageSwitcher.js

```diff
@@ -1,39 +1,32 @@
-import React from 'react'
+import * as React from 'react'
```

Comment:
Do we need to import everything from react can you we optimize this?

Reply:
Actually using `import * as React` is an improved optimization over `import React`.

Webpack will rewrite `import * as React` into `import { useState, createElement, etc. }  from 'react'` when it compiles the file. Using `import React` means it will always import the whole module after compilation, even if everything isn't used.